tests/ui/arch/all_memory_barrier.stderr

```diff
@@ -1,7 +1,14 @@
+error: error:0:0 - failed spawn executable: No such file or directory (os error 2)
```

Comment:
We don't want this error to be here :P Do you have `spirv-val` installed and in your $PATH?